README.md

```diff
@@ -141,7 +161,9 @@ Go to http://bin.mailgun.net. The Postbin will generate a special URL. Save that
 
 ```php
 # First, instantiate the SDK with your API credentials and define your domain. 
-$mg = new Mailgun('key-example', 'bin.mailgun.net', 'aecf68de', $ssl = False);
+$mg = new Mailgun('key-example', null, 'bin.mailgun.net');
+$mg->setApiVersion('aecf68de');
+$mg->setSslEnabled('false');
```

Comment:
This is the [existing example](https://github.com/mailgun/mailgun-php#debugging) of how to debug by posting the output to pastebin. This should later be updated, example by using the [Logger plugin](https://github.com/php-http/plugins/blob/master/src/LoggerPlugin.php).